modules/ROOT/pages/to-share-api-asset-to-portal.adoc

```diff
@@ -4,6 +4,10 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
+An organization's Exchange administrators control whether each API asset created in the organization is shared in the public portal. Set which users have the Exchange administrator role in the Access Management > Roles menu, and find more information in xref:to-set-permissions.adoc[Set User Access Permissions].
+
+== Add an Asset to a Public Portal
+
```

Comment:
You could add a sentence or two here about the circumstances under which your audience might want or need to do this.